### DISCUSSION TITLE
❗️❗️ The dist folder has been removed

### DISCUSSION THREAD

**Discussion by sampotts** [sampotts/plyr#2094] [General]
To prevent the common issues with PRs including the built files for a change, I've decided to remove and ignore the `dist` directories. If you need access to it, you can do so via services like unpkg:

https://unpkg.com/browse/plyr/dist/
or for a specific version:
https://unpkg.com/browse/plyr@3.6.2/dist/

Cheers 👍

**Comment by skerbis** [comment]
There is no Download-Button or something like that. 
I've also tried building it via yarn but it crashes without reasons.

**Reply by sampotts** [reply]
`yarn` and then `yarn build` should do it? Hard to debug if there's no error though.

**Reply by skerbis** [reply]
[15:06:18] The following tasks did not complete: build, <parallel>, css, css:plyr.css, css:demo.css, css:error.css
[15:06:18] Did you forget to signal async completion?

**Reply by skerbis** [reply]
ok installed it now via npm

**Comment by alirezadigi** [comment]
Hi @sampotts !
How can we create dist folder with its content?(without using unpkg?)

**Reply by skerbis** [reply]
i've used: npm install plyr

**Reply by alirezadigi** [reply]
@skerbis Thanks a lot!
I used `npm install plyr` in an empty directory.It works.
If it didn't work try `npm install --prefix . plyr`.

**Comment by bybpow** [comment]
Is there a way to get these ready-to-go unpacked files from a release tag?
